Sources/BraintreeCore/BTHTTP.m

```diff
@@ -110,7 +112,11 @@ - (void)GET:(NSString *)aPath completion:(void(^)(BTJSON *body, NSHTTPURLRespons
 }
 
 - (void)GET:(NSString *)aPath parameters:(NSDictionary *)parameters completion:(void(^)(BTJSON *body, NSHTTPURLResponse *response, NSError *error))completionBlock {
-    [self httpRequest:@"GET" path:aPath parameters:parameters completion:completionBlock];
+    if ([aPath containsString:@"configuration"]) {
```

Comment:
What about adding a `GET cache:TRUE....` method instead of using this branching logic? Then future GETS could leverage it.

Reply:
Yes good idea - updated this to have a `shouldCache` param and made the internal method names more generic 👍